feeds/trilliumtransit.com.dmfr.json

```
@@ -2522,6 +2522,12 @@
     },
     {
       "spec": "gtfs",
+      "id": "f-9q7m-portervilletransit",
+      "urls": {
+        "static_current": "http://data.trilliumtransit.com/gtfs/porterville-ca-us/porterville-ca-us.zip"
+      }
+    },
+    {
```

@rpedraza01 try adding this missing part:
```suggestion
    {
      "spec": "gtfs",
```